src/plays/context-with-real-usecases/ContextWithRealUsecases.jsx

```diff
@@ -0,0 +1,88 @@
+import { createContext, useEffect, useState } from "react";
+import PlayHeader from "common/playlists/PlayHeader";
+import "./contextWithRealUsecases.css";
+import ContextInfoSlider from "./components/ContextInfoSlider";
+import Heading from "./components/Heading";
+import Menu from "./components/Menu";
+import Main from "./components/Main/Main";
+import Recipes from "./Recipes";
+export const cusineContext = createContext();
+
+function ContextWithRealUsecases(props) {
+  // Your Code Start below.
+
+  const [activeCuisine, setActiveCuisine] = useState(null);
+
+  //slide index
+  const [index, setIndex] = useState(0);
+
+  const activeCuisineHandler = (cuisine) => {
+    setActiveCuisine(cuisine);
+  };
+
+  //get an array of cuisines
+  const cuisines = Recipes.map((item) => {
+    return item.recipe.cusine;
+  });
+
+  //remove duplicate cuisines.
+  const uniqCuisines = [...new Set(cuisines)];
+
+  //get recipes filtered by cuisine type
+  const recipesByCusine = Recipes.filter(
+    (item) => item.recipe.cusine === activeCuisine
```

Comment:
@atapas,  if you look at below the above code. there is a closing parenthesis right after the above code. Hence the code is not complete and I can't have a semicolon before closing parenthesis. Please do not worry about these issues as I have prettier it will take care of formatting issues.